utils/build.gradle

```diff
@@ -15,4 +15,5 @@ dependencies {
 
     compile "commons-codec:commons-codec:$commonsCodecVersion"
     compile "org.apache.commons:commons-compress:$commonsCompressVersion"
+    compile "org.tukaani:xz:$xzVersion"
```

Comment:
Is the 7zip support of commons compress too limitied, or why can't we use it?

Reply:
Is this used at all? I don't see any imports from `org.tukaani` below.

Reply:
The Apache Commons implementation [actually depends on it](https://stackoverflow.com/a/28277555/1127485), but as it's optional if you're not using 7-Zip, they did not add `xz` as a dependency...